src/components/Library/LibraryForm/LibraryForm.tsx

```diff
@@ -13,30 +14,32 @@ export default function LibraryForm() {
     pages: '',
   };
 
+  const { t } = useTranslation();
+
   const formik = useFormik({
     initialValues,
     validationSchema: yup.object({
       name: yup
         .string()
-        .min(3, 'Min 3 symbols')
-        .max(50, 'Max 50 symbols')
-        .required('Required'),
+        .min(3, 'validation.min3')
+        .max(50, 'validation.max50')
+        .required('validation.required'),
       author: yup
         .string()
-        .min(3, 'Min 3 symbols')
-        .max(40, 'Max 40 symbols')
-        .required('Required'),
+        .min(3, 'validation.min3')
+        .max(40, 'validation.max54')
+        .required('validation.required'),
       year: yup
         .string()
```

Comment:
shold be number? not string

Reply:
This because of ts, there are validation on number, so dont worry

Reply:
ok:)